ui/src/main/webapp/src/app/reports/technologies/technologies-report-ejb.component.html

```diff
@@ -0,0 +1,189 @@
+                                { title: 'Implementation', isSortable: true, sortBy: 'class'},
+                                { title: 'JNDI Location', isSortable: true, sortBy: 'location'}
+                            ]">
+                        </thead>
+                        <tbody>
+                        <tr *ngFor="let value of this.sortedEjbSessionStatefulBean">
+                            <td class="col-md-2">{{value.name}}</td>
+                            <td class="col-md-2">{{value.interface}}</td>
+                            <td class="col-md-6"><a [routerLink]="['../source/' + value.sourceVertexId]">{{value.class}}</a></td>
+                            <td class="col-md-2">{{value.location}}</td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </wu-all-data-filtered-message>
+            </div>
+        </div>
+
+        <div class="panel panel-default panel-primary wuTechnologies">
+            <div class="panel-heading">
+                <h3 class="panel-title">Entity Beans</h3>
+            </div>
+            <div class="panel-body">
+                <wu-all-data-filtered-message
+                        [filteredItems]="filteredEjbEntityBean"
+                        [unfilteredItems]="ejbEntityBean"
+                        (clearFilter)="clearSearch()">
+                    <table class="table table-bordered table-hover" *ngIf="filteredEjbEntityBean.length > 0; else noDataAvailable">
+                        <thead wu-sortable-table
+                               [(sortedData)]="sortedEjbEntityBean" [data]="filteredEjbEntityBean"
+                               [tableHeaders]="[
+                                { title: 'Bean Name', isSortable: true, sortBy: 'name' },
+                                { title: 'Interface', isSortable: true, sortBy: 'interface'},
+                                { title: 'Implementation', isSortable: true, sortBy: 'class'},
+                                { title: 'JNDI Location', isSortable: true, sortBy: 'location'}
+                            ]">
+                        </thead>
+                        <tbody>
+                        <tr *ngFor="let value of this.sortedEjbEntityBean">
+                            <td class="col-md-2">{{value.name}}</td>
+                            <td class="col-md-2">{{value.interface}}</td>
+                            <td class="col-md-6"><a [routerLink]="['../source/' + value.sourceVertexId]">{{value.class}}</a></td>
+                            <td class="col-md-2">{{value.location}}</td>
+                        </tr>
+                        </tbody>
+                    </table>
+
+                </wu-all-data-filtered-message>
+            </div>
+        </div>
+    </ng-container>
+</div>
+
+<ng-template #noDataAvailable>
+    <span>There are no beans of this type.</span>
+</ng-template>
```

Comment:
This will appear here if any of the tables are empty.
It should either be one for each table. Or - doesn't the ` +                <wu-all-data-filtered-message ` handle this?